src/core/BufferGeometry.js

```diff
@@ -668,11 +668,11 @@ class BufferGeometry extends EventDispatcher {
 
 			if ( index ) {
 
-				for ( let i = 0, il = index.count; i < il; i += 3 ) {
+				for ( let i = 0, il = index.count; i < il; i ++ ) {
```

Comment:
I'm afraid this change isn't right. An entry in the index buffer attribute represents a single index which has an item size of `1` (not `3` like a position or normal attribute).